webui/react/src/hooks/useFeature.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react';
+import React, { useEffect, useMemo } from 'react';
+
+import StoreProvider, { initInfo, StoreAction, useStoreDispatch } from 'contexts/Store';
+
+import useFeature from './useFeature';
+
+const FeatureTest: React.FC = () => {
+  const storeDispatch = useStoreDispatch();
+  const feature = useFeature();
+  const testInfo = useMemo(() => ({ ...initInfo, featureSwitch: ['webhooks'] }), []);
+  useEffect(() => {
+    storeDispatch({ type: StoreAction.SetInfo, value: testInfo });
+  }, [storeDispatch, testInfo]);
+
+  return (
+    <ul>
+      <li>{feature.isOn('trials_comparison') && 'trials_comparison'}</li>
```

Comment:
So that this isn't reliant on the current set of feature switches, you can do a cast here:
```typescript
const feature1 = 'feature1' as ValidFeature
const feature2 = 'feature2' as ValidFeature
```
Then you can test if feature1 is on and feature2 is off.

Reply:
Great!